src/js/components/TaskListComponent.jsx

```diff
@@ -167,10 +170,7 @@ var TaskListComponent = React.createClass({
                 </th>
             </tr>
           </thead>
-          <PagedContentComponent
```

Comment:
Not sure what is going on here? Why do we not want to use the `PagedConentComponent`? Does it not provide the needed functionality?